services/web/server/tests/unit/with_dbs/03/resource_usage/test_usage_services__export.py

```diff
@@ -0,0 +1,115 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+# pylint: disable=too-many-arguments
+# pylint: disable=too-many-statements
+
+
+import json
+from collections.abc import Iterator
+from typing import cast
+
+import pytest
+import sqlalchemy as sa
+from aiohttp import web
+from aiohttp.test_utils import TestClient
+from models_library.resource_tracker import ServiceResourceUsagesFilters
+from models_library.rest_ordering import OrderBy
+from pydantic import AnyUrl, parse_obj_as
+from pytest_mock.plugin import MockerFixture
+from pytest_simcore.helpers.utils_login import UserInfoDict
+from simcore_postgres_database.models.wallets import wallets
+from simcore_service_webserver.db.models import UserRole
+
+
+@pytest.fixture
+def mock_export_usage_services(mocker: MockerFixture) -> tuple:
+    mock_export_usage = mocker.patch(
+        "simcore_service_webserver.resource_usage._service_runs_api.service_runs.export_service_runs",
+        spec=True,
+        return_value=parse_obj_as(AnyUrl, "https://www.google.com/"),
+    )
+    return mock_export_usage
+
+
+@pytest.fixture()
+def setup_wallets_db(
+    postgres_db: sa.engine.Engine, logged_user: UserInfoDict
+) -> Iterator[int]:
+    with postgres_db.connect() as con:
+        result = con.execute(
+            wallets.insert()
+            .values(
+                name="My wallet",
+                owner=logged_user["primary_gid"],
+                status="ACTIVE",
+                product_name="osparc",
+            )
+            .returning(sa.literal_column("*"))
+        )
+        row = result.fetchone()
+        yield cast(int, row[0])
+        con.execute(wallets.delete())
+
+
+@pytest.mark.parametrize(
+    "user_role,expected",
+    [
+        (UserRole.ANONYMOUS, web.HTTPUnauthorized),
+        (UserRole.GUEST, web.HTTPForbidden),
+        (UserRole.USER, web.HTTPOk),
+        (UserRole.TESTER, web.HTTPOk),
+        (UserRole.PRODUCT_OWNER, web.HTTPOk),
+        (UserRole.ADMIN, web.HTTPOk),
+    ],
+)
```

Comment:
please consider using and extending ```standard_role_response``` in packages/pytest-simcore/src/pytest_simcore/helpers/utils_webserver_unit_with_db.py

Reply:
Considered and ignored for now, I am sorry but can not spend whole day on each improvement. I focused on the MAJOR ones :+1: